src/main/scala/ai/privado/audit/AuditReportConstants.scala

```diff
@@ -120,4 +122,10 @@ object AuditReportConstants {
 
   val JS_ELEMENTS_TO_BE_EXCLUDED =
     "^(?i)(modules?|loggers?|val|console|require|_|get|key|value|data|page|url|Set|filter|<init>|err|errors|axios|express|router|component|Instance|utils?|app|undefined|context|process|...props?|async|await|const|let|var|this)$"
+
+  val URLREGEX = ".*(://).*"
```

Comment:
PRIVADO_URL_SERVER="privado.ai/urls"

Reply:
Also include all literals which have `/` in them 
cc - @HiteshMah-Jan

Reply:
okay, i will include all the literal having `/` in it.

Reply:
done @khemrajrathore @HiteshMah-Jan please take a look.